docs/transparent-validation/protectli/laboratory-assembly-guide.md

```diff
@@ -212,6 +250,29 @@ The method of setting and using serial connection is described in the
         rte_ctrl rel
         ```
 
+=== "VP66xx"
+
+    Power supply controlling (in this case: controlling the state of Sonoff)
+    should be performed based on the `sonoff.sh` script implemented in
+    `meta-rte` (OS image dedicated to the RTE platform).
+
+    > Note, that before using the above-mentioned script, it should be modified and
+    `SONOFF_IP` parameter should be set in accordance with obtained Sonoff IP address.
+
+    To perform basic power operations use the below-described commands:
+
+    1. Turn on the power supply:
+
+        ```bash
+        ./sonoff on
```

Comment:
But such script does not exist.

We can simply refer to the OSFV CLI: https://github.com/Dasharo/osfv-scripts/tree/main/osfv_cli - all actions are documented there.

Reply:
I copied it from VP46xx - fixed it for both platforms

Reply:
Changed for vp46xx too in: 5084c58dada6befae99d63f2a81cb35ccb4a4962